app/controllers/application_controller.rb

```diff
@@ -138,6 +138,47 @@ def render_unsupported_format
     end
   end
 
+  # Determines the source of the action based on the relationship between logged_in user and applicant
+  #
+  # @param person [Person] The person applying for financial assistance
+  # @param logged_in_user [User] The current user making the request
+  # @return [Symbol] The source of the action (:user, :admin, :broker, :assister, or :unknown)
+  #
+  # @note This method applies for the Individual market only.
+  def fetch_origin_source(person, logged_in_user)
```

Comment:
- The term origin_source is redundant, as "origin" and "source" are similar in meaning.

@saikumar9  can this be renamed to fetch_origin or fetch_source?
```suggestion
  def fetch_source(person, logged_in_user)
  ```
  or
  
  ```suggestion
  def fetch_origin(person, logged_in_user)
```